spanner/spanner_snippets/spanner/spanner_insert_graph_data.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_insert_graph_data]
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"cloud.google.com/go/spanner"
+)
+
+func parseTime(rfc3339Time string) time.Time {
+	t, _ := time.Parse(time.RFC3339, rfc3339Time)
+	return t
+}
+
+func insertGraphData(w io.Writer, db string) error {
+	ctx := context.Background()
+	client, err := spanner.NewClient(ctx, db)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+
+	personColumns := []string{"id", "name", "birthday", "country", "city"}
+	accountColumns := []string{"id", "create_time", "is_blocked", "nick_name"}
+	ownColumns := []string{"id", "account_id", "create_time"}
+	transferColumns := []string{"id", "to_id", "amount", "create_time", "order_number"}
+	m := []*spanner.Mutation{
```

Comment:
The insert statements are standard insert SQL DML statements, but these samples are useful because of the schema of the database with the property graph is different. This sample would not be runnable with any of the other example SQL databases shows in other samples.

Reply:
Hmm, okay. I would expand on the comment to make it clearer how this set of inserts pertains to the specific property graph database--they're not just like any other inserts.